src/data/set/basic.lean

```diff
@@ -843,6 +843,8 @@ end
 section sep
 variables {p q : α → Prop} {x : α}
 
+theorem sep_eq_inter : {x ∈ s | p x} = s ∩ {x | p x} := rfl
```

Comment:
This surely exists already, I think it's `set_of_inter`?

Reply:
https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.set_of_inter_eq_sep